server-side/server_handler.py

```diff
@@ -0,0 +1,44 @@
+""" Getter and Setter class for fetching hostname and IP."""
+
+import socket
+import psutil
+
+
+class ServerHandle:
+    """class with getter methods returning various system variables."""
```

Comment:
detailed explanation please- which system variables

Reply:
```suggestion
    """Root method which returns with Server hostname.
```

This is done, added a little more info to the DocString.